test/fixtures/locations.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "location": {
+      "id": 1059367776,
+      "name": "Alpha Location"
+    },
+    "message": "Current location.",
+    "movable": false
+  },
```

Comment:
This fixture isn't just locations, it includes some movable info. Better named `fulfillment_order_move_locations`